_data/simple-icons.json

```diff
@@ -8495,9 +8495,9 @@
         },
         {
             "title": "Ko-fi",
-            "hex": "FF5E5B",
-            "source": "https://more.ko-fi.com/brand-assets",
-            "guidelines": "https://more.ko-fi.com/brand-assets"
+            "hex": "FC5A17",
+            "source": "https://storage.ko-fi.com/cdn/brandasset/v2/kofi_symbol.png",
+            "guidelines": "https://help.ko-fi.com/hc/en-us/articles/360007937553-Ko-fi-Content-Guidelines"
```

Comment:
Let's convert the EPS assets from the branding page so we have a more accurate SVG than a traced one.

Reply:
It was converted using the trace bitmap function of inksacape.
What is EPS? do you have a tool for that?